### PR TITLE
[Redhat Appstudio e2e] : Add Quay tokens required for release tests

### PR DESCRIPTION
**Pull request by sawood14012**
Adding tokens required for new release tests as secrets
Signed of by msawood@redhat.com

### CODE REVIEW

**Review by jkopriva — APPROVED** [review]

File: ci-operator/config/redhat-appstudio/build-service/redhat-appstudio-build-service-main.yaml [hunk 1]
```diff
@@ -35,6 +35,8 @@ tests:
         export GITHUB_USER=$(cat /usr/local/ci-secrets/redhat-appstudio-qe/github-user)
         export GITHUB_TOKEN=$(cat /usr/local/ci-secrets/redhat-appstudio-qe/github-token)
         export QUAY_TOKEN=$(cat /usr/local/ci-secrets/redhat-appstudio-qe/quay-token)
+        export QUAY_OAUTH_TOKEN_RELEASE_BUILD=$(cat /usr/local/ci-secrets/redhat-appstudio-qe/quay-oauth-token-release-build)
+        export QUAY_OAUTH_TOKEN_RELEASE_RELEASE=$(cat /usr/local/ci-secrets/redhat-appstudio-qe/quay-oauth-token-release-release)
```
Comment (psturc) [inline ci-operator/config/redhat-appstudio/build-service/redhat-appstudio-build-service-main.yaml:39]: Afaik there's no need to add them to build-service config. This job doesn't run any "release-specific" test

Comment (kasemAlem) [reply ci-operator/config/redhat-appstudio/build-service/redhat-appstudio-build-service-main.yaml:39]: lt's just a naming, 
you are right will rename it according to repo 
Talked to Meer and the naming will be as below and the tokens will be in release-service e2e-job :
QUAY_OAUTH_TOKEN_RELEASE_SOURCE
QUAY_OAUTH_TOKEN_RELEASE_DESTINATION

**Comment by kasemAlem** [comment]
@sawood14012 , 
all looks fine by me 
@psturc, what do you say ?

**Comment by psturc** [comment]
I think we now need an approval from someone from [these owners](https://github.com/openshift/release/blob/master/ci-operator/config/redhat-appstudio/infra-deployments/OWNERS) now

**Review by flacatus — APPROVED** [review]

**Comment by sawood14012** [comment]
> Details

**Pull request closed without merging**